redhat-cop/casl-ansible
# Docker volume storage refactor

#### What does this PR do?
Allows for the provisioning of a cluster without using extra volumes for storing containers.

#### How should this be manually tested?
Provision a cluster and ensure that `openshift_openstack_ephemeral_volumes` is set to true within your inventory.

#### Who would you like to review this?
cc: @redhat-cop/casl
